src/bankAccount.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, initial) {
```

Comment:
what does `initial` mean?
lastname?
age?
country?

```suggestion
function BankAccount(name, money) {
```